### ISSUE TITLE
Double headers showing in the module

### ISSUE THREAD

**Issue by wretanhewe** [kjb085/MMM-Reddit#18]
Hello, I'm using the default config content for headlines, but i am always seeing an extra header at the top of the instance of the module. Is there a way to remove this?
This is what I see on the mirror in the module, and what is in my config (for troubleshooting, direct copy/paste from the main page of this project)

![Annotation 2020-04-03 132115](https://user-images.githubusercontent.com/63121332/78393307-2349ff00-75af-11ea-92ad-5e928d9f24e3.jpg)
![Annotation 2020-04-03 132916](https://user-images.githubusercontent.com/63121332/78393309-23e29580-75af-11ea-9021-0011f5989104.jpg)

**Comment by JoMa25** [comment]
Same, I got wto times "Hot Posts from r/aww. Doenst look that nice

**Comment by larryare** [comment]
Same.

**Comment by kjb085** [comment]
Looking into this

**Comment by JohnTrunix** [comment]
Any solutions? I have the same problem.. Thx

**Comment by kjb085** [comment]
Can anyone list what other modules you guys have running? I'm not running into this same issue during testing.

**Comment by JohnTrunix** [comment]
I just have some of the default modules and your module installed and activated in the config.js file.

**Comment by JohnTrunix** [comment]
I just have some of the default modules and your module installed and activated in the config.js file.

And here my config file for your module, i think its the default one:

{
			module: "MMM-Reddit",
			position: "bottom_center",
			config: {
				subreddit: "earthporn",
				type: "hot",
				displayType: "image",
				imageQuality: "high",
				count: 10,
				show: 1,
				width: 500,
				updateInterval: 5,
				rotateInterval: 30,
				showHeader: true,
				headerType: "sentence",
				
			}
		},

**Comment by uffsalot** [comment]
> Can anyone list what other modules you guys have running? I'm not running into this same issue during testing. Also what version of MM2 is everyone running, I'm still on 2.6, wondering if that might be part of the problem as well.

I am using MagicMirror v2.11.0.

Here is a list of my used plugins:

- alert
- MMM-Fuel
- updatenotification
- clock
- calendar
- currentweather
- weatherforecast
- MMM-PublicTransportHafas
- mmm-hue-lights
- MMM-Reddit


MMM-Reddit config:
`{
            module: "MMM-Reddit",
            position: "top_center",
            config: {
                subreddit: 'SupermodelCats',
                displayType: 'image',
                imageQuality: 'high',
                count: 10,
                show: 1,
                width: 300,
                showAll: false,
                showRank: false,
                showScore: true,
                showAuthor: true,
                showSubreddit: false,
                showNumComments: false,
                showGilded: false,
            }
        },`


![2020-06-17 21_06_55-MagicMirror²](https://user-images.githubusercontent.com/1371511/84939474-aecb2a80-b0de-11ea-8660-90c8b55b115a.png)

**Comment by kjb085** [comment]
Just upgraded to the latest version of MM and I'm seeing the issue now, so I have something to work with finally

**Comment by kjb085** [comment]
Yall can pull the latest and see if that fixes your problem - let me know if the issue persists. Core issue was a bug in the MagicMirror code that I had circumvented, but left code to make it function once the bug was addressed as well. I'm looking forward to people on older versions of MM2 positing issues that the header doesn't show up now. It had been so long that I forgot all about this unfortunately.
